azure/scripts/playbooks/roles/ocpcluster/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+---         
+- name: "Download OpenShift Binaries"
+  ansible.builtin.shell: |
```

Comment:
You should avoid using `ansible.builtin.shell` or `ansible.builtin.command` for most commands like this. This defeats the purpose of ansible as there are native ansible modules that handle many of these tasks.

Reply:
This is actioned.